firstvoices/backend/models/widget.py

```diff
@@ -64,9 +64,9 @@ class Meta:
         verbose_name_plural = _("site widgets")
         rules_permissions = {
             "view": predicates.is_visible_object,
-            "add": predicates.is_language_admin_or_super,
-            "change": predicates.is_language_admin_or_super,
-            "delete": predicates.is_language_admin_or_super,
+            "add": predicates.can_add_controlled_data,
+            "change": predicates.can_edit_controlled_data,
+            "delete": predicates.can_delete_controlled_data,
```

Comment:
I believe we also need to update the permissions on the SiteWidgetList... possibly should be using the equivalent ones for uncontrolled data. @NolanVH do you have input on this?

Reply:
We probably don't need to since SiteWidgetList, and SiteWidgetListOrder models are automatically handled by the Site and SitePage instances. The only time somebody would need permission to change those models are in the admin panel.

As an aside, I believe only language admins and above should be allowed to create or edit widgets based on the user roles doc and it looks like this change would give roles below language admin the ability to do this.

Reply:
Oh good call. Pages and Widgets should both use be is_language_admin_or_super